Plugins/Json/include/Acts/Plugins/Json/GeometryHierarchyMapJsonConverter.hpp

```diff
@@ -102,9 +104,35 @@ class GeometryHierarchyMapJsonConverter {
 
 // implementations
 
-template <typename value_t>
-nlohmann::json GeometryHierarchyMapJsonConverter<value_t>::toJson(
-    const Container& container) const {
+// auxiliary struct to indicate a missing specialization of a template which
+// requires specialisation
+template <typename T, class decorator_t>
+struct missing_specialization {
```

Comment:
Makes sense to use standard constructs. I tend to reimplement simple constructs, since it requires less knowledge when reading the code, but I guess it leaves more room for inconsistent or confusing behaviour. Anyway done.